go.mod

```diff
@@ -21,34 +21,60 @@ require (
 	github.com/aws/aws-sdk-go v1.13.20
 	github.com/aws/aws-xray-sdk-go v1.0.0-rc.5 // indirect
 	github.com/basvanbeek/ocsql v0.0.0-20180627133915-d946a62502e3
+	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
 	github.com/census-ecosystem/opencensus-go-exporter-aws v0.0.0-20180411051634-41633bc1ff6b
+	github.com/coreos/bbolt v1.3.1-coreos.6 // indirect
+	github.com/coreos/etcd v3.3.9+incompatible
```

Comment:
Is it the same thing as `go.etcd.io/etcd`? Just curious, but this seems to be a transitive dependency though.

Reply:
It looks like they are moving everything to go.etcd.io/etcd, but not everything has had a release with that updated yet, so we're pulling in both. I updated to use the old path consistently for now.